tools/wpt/browser.py

```diff
@@ -131,6 +143,12 @@ def find_certutil(self):
     def find_webdriver(self):
         return find_executable("geckodriver")
 
+    def _get_executive_path(self, path, exe):
```

Comment:
I'm not sure that making this a method rather than a function is an improvement. It's not like what it does is specific to this particular class.

Reply:
This (along with its caller) has been removed.